### PR TITLE
Add infoblox host variable

### PR DESCRIPTION
**Pull request by bp85**
- In all cases infoblox host is not the dns master

### CODE REVIEW

**Comment by bp85** [comment]
once this is approved, I'll change https://github.com/theforeman/puppet-foreman_proxy/blob/master/manifests/plugin/dns/infoblox.pp accordingly so that existing workflows doesn't break

**Comment by dmitri-d** [comment]
Could you updated tests please? In particular, configuration_test.rb requires changes.

**Comment by bp85** [comment]
@witlessbird is there anything else need to be updated?

File: lib/smart_proxy_dns_infoblox/plugin_configuration.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ def load_dependency_injection_wirings(container_instance, settings)
                                       ::Infoblox.wapi_version = '2.0'
                                       ::Infoblox::Connection.new(:username => settings[:username],
                                                                  :password => settings[:password],
-                                                                 :host => settings[:dns_server],
+                                                                 :host => settings[:infoblox_host],
```
Comment (dmitri-d) [inline lib/smart_proxy_dns_infoblox/plugin_configuration.rb]: I think ```:infoblox_host``` should be an optional parameter and only used when set. In the absence of ```:infoblox_host```, the value in ```:dns_server``` setting should be used.

**Comment by ekohl** [comment]
Could you rebase this? It'd make review easier.

**Comment by lzap** [comment]
This needs to be rebased it's a mess, thanks!

**Comment by lzap** [comment]
I am ready to help if you want to continue, for now I am closing the issue. Thanks for your contribution so far!

**Comment by bp85** [comment]
@lzap sorry I totally missed previous comments!
I just rebased and opened a new PR https://github.com/theforeman/smart_proxy_dns_infoblox/pull/25.
Any suggessions are welcome.

**Pull request closed without merging**